lib/elixir/lib/access.ex

```diff
@@ -137,7 +137,8 @@ defmodule Access do
 
   """
 
-  @type t :: list | map | nil | any
+  @type container :: keyword | struct | map | any
```

Comment:
I think it is. at least with tuples.
https://github.com/elixir-lang/elixir/pull/5311